src/mbgl/map/map_data.hpp

```diff
@@ -49,24 +50,25 @@ class MapData {
     std::vector<std::string> getClasses() const;
 
 
-    inline bool getDebug() const {
-        return debug;
-    }
-    inline bool toggleDebug() {
-        return debug ^= 1u;
-    }
-    inline void setDebug(bool value) {
-        debug = value;
+    inline MapDebugMode getDebug() const {
+        return debugMode;
     }
 
-    inline bool getCollisionDebug() const {
-        return collisionDebug;
+    inline void toggleDebug() {
```

Comment:
Can we rename `toggleDebug` to reflect what it actually does? It's not really toggling rather than cycling through the options.